### PR TITLE
Add VBScript engine

### PR DESCRIPTION
**Pull request by fviale**
- based on the cscript command

Also in this PR:
 - add spotless and licence header
 - add "args" binding to all script engines command line arguments
 - upgrade engine version

### CODE REVIEW

File: src/main/java/jsr223/nativeshell/NativeShellRunner.java [hunk 1]
```diff
@@ -81,19 +112,35 @@ public int run(String command, ScriptContext scriptContext) throws ScriptExcepti
...
+                   scriptContext.getErrorWriter(),
+                   processTreeKiller);
+    }
+
+    private List<String> getArguments(ScriptContext scriptContext) {
+        Bindings bindings = scriptContext.getBindings(ScriptContext.ENGINE_SCOPE);
+        if (bindings != null && bindings.containsKey(ARGS)) {
+            if (bindings.get(ARGS) instanceof String[]) {
+                String[] arguments = (String[]) bindings.get(ARGS);
+                return Arrays.asList(arguments);
```
Comment (Mykhailenko) [inline src/main/java/jsr223/nativeshell/NativeShellRunner.java:138]: just to confirm, `Arrays.asList` returns array which pretends to be a List

File: src/main/java/jsr223/nativeshell/NativeShellScriptEngine.java [hunk 2]
```diff
@@ -1,13 +1,38 @@
...
 
-import org.ow2.proactive.scheduler.common.SchedulerConstants;
-
 import java.io.Reader;
 import java.io.Serializable;
 import java.util.Map;
 
 import javax.script.*;
 
+import org.ow2.proactive.scheduler.common.SchedulerConstants;
```
Comment (Mykhailenko) [inline src/main/java/jsr223/nativeshell/NativeShellScriptEngine.java:34]: useless import?

File: src/main/java/jsr223/nativeshell/vbs/VbsScriptEngineFactory.java [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+    @Override
+    public Object getParameter(String key) {
+        return parameters.get(key);
+    }
+
+    @Override
+    public String getMethodCallSyntax(String obj, String m, String... args) {
+        String methodCall = m + " ";
+        for (String arg : args) {
```
Comment (Mykhailenko) [inline src/main/java/jsr223/nativeshell/vbs/VbsScriptEngineFactory.java]: `String methodCall = m + " " + String.join(" ", args);`

**Review by Mykhailenko — APPROVED** [review]

**Comment by fviale** [comment]
Retest this

File: src/main/java/jsr223/nativeshell/NativeShellRunner.java (hunk 1 above)
Comment (fviale) [reply src/main/java/jsr223/nativeshell/NativeShellRunner.java:138]: It does not matter, as long as it is a collection which can be added to the command

File: src/main/java/jsr223/nativeshell/NativeShellScriptEngine.java (hunk 2 above)
Comment (fviale) [reply src/main/java/jsr223/nativeshell/NativeShellScriptEngine.java:34]: no it is used inside this class

File: src/main/java/jsr223/nativeshell/vbs/VbsScriptEngineFactory.java (hunk 3 above)
Comment (fviale) [reply src/main/java/jsr223/nativeshell/vbs/VbsScriptEngineFactory.java]: right, gonna replace all occurences of this ugly code

**Pull request merged**